Code/PgSQL/rdkit/Makefile

```diff
@@ -61,22 +61,23 @@ PG_CONFIG  = pg_config
 MODULE_big = rdkit
 OBJS       = rdkit_io.o mol_op.o bfp_op.o sfp_op.o rxn_op.o rdkit_gist.o bfp_gist.o bfp_gin.o low_gist.o guc.o cache.o adapter.o bitstring.o
 PGXS       := $(shell $(PG_CONFIG) --pgxs)
-PG91 = $(shell $(PG_CONFIG) --version | grep -qE " 8\.| 9\.0" && echo no || echo yes)
+PG95 = $(shell $(PG_CONFIG) --version | grep -qE " 9\.[1-4]" && echo no || echo yes)
```

Comment:
Yes, sorry, this could indeed deserve some discussion. version 9.0 reached EOL almost one year ago (Sep 2015) and the upstream support for it has ended. If necessary, the compatibility with pre-9.1 versions could be restored, but I thought it could be a good occasion for a cleanup.